xml/obs/systemsmanagement:Agama:Devel.xml

```diff
@@ -2,9 +2,13 @@
     project_pattern="systemsmanagement:Agama:Devel"
     dist_path="obspublish-other::openqa/systemsmanagement:Agama:Devel/"
     distri="opensuse"
-    version="agama-9.0"
-    archs="x86_64 aarch64 ppc64le s390x">
-    <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/">
-        <hdd filemask="agama-installer.*-openSUSE-Build.*iso$"/>
-    </flavor>
+    version="agama-9.0">
+    <batch name="base" archs="86_64 aarch64 ppc64le">
+        <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/">
+            <hdd filemask="agama-installer.*-openSUSE-Build.*iso$"/>
```

Comment:
Do you really need hdd? I expected it should work without it as well

Reply:
I updated it following your suggestion, I think this will be more simple and clear. :)